packages/flame/lib/src/game/game_widget/game_widget.dart

```diff
@@ -71,6 +71,9 @@ class GameWidget<T extends Game> extends StatefulWidget {
   final MouseCursor? mouseCursor;
   final List<String>? initialActiveOverlays;
 
+  /// Whether the game should assume the behavior of a [RepaintBoundary].
+  ///
+  /// Defaults to `true`.
```

Comment:
Are these the docs mentioned in the commit? 😄 
We should probably add a short paragraph about it in the real docs too.

Reply:
I didn't find a proper place to add it on the MD files. Also, we don't doc most of the features of the game widget anywhere besides the dart docs.

Reply:
This will be fixed in #2344